### ISSUE TITLE
Starting Packages

### ISSUE THREAD

**Issue by Butterfly-Dragon** [Orcpub/orcpub#111]
**Is your feature request related to a problem? Please describe.**

I am currently unable to edit/add/remove the contents of a "starting package" on the character sheet without manually editing the character sheet.

Also i would like to be able to select a starting package from the items menu and have all the constituent items be available.

**Describe the solution you'd like**
• whenever a starting package is part of a character creation to have a "finalize button" that breaks it down into its constituent items, rather than "just" listing them in the charsheet. This way when items gets removed/used i can edit the sheet on orcpub rather than doing manual edits.
• whenever a starting package is selected from the items list, to have a button to break it down
• use a special menu instead of the item menu for starting packages so i can see their listings and chose which i want before "breaking it down" into constituent items.

**Describe alternatives you've considered**
fuzzy solutions that honestly seemed harder to implement, like editing stuff on the summary side or finalizing every beginning choice (which would still be good but is of grander scope).

**Additional context**
current situation:
![image](https://user-images.githubusercontent.com/33538906/54682603-abc64080-4b0f-11e9-8a3e-b17a7a636a16.png)

**Comment by datdamnzotz** [comment]
related to #138

**Comment by Butterfly-Dragon** [comment]
Yeah i can see the relation. At this point i'd suggest that once initial equipment is chosen a checkbox should appear to "finalize it" and add them as normal items, and once a "package" is among the items there is a button to "unpack the package". It's... an hassle, but it's better than nothing 🤔😕😖
